src/plugins/template/backend/int_executable.cpp

```diff
@@ -113,8 +88,6 @@ bool ov::runtime::interpreter::INTExecutable::call(std::vector<ov::Tensor>& outp
             results_map.emplace(output, output_count);
     }
 
-    auto overrider = TemporaryOverrideOutputs(m_model, tensor_map);
```

Comment:
Is it really not used?
It looks like at end of scope the dtor will restore the original shapes.

Reply:
you're right.
Rewrote it to make code more readable.